### ISSUE TITLE
Unsupported major.minor version 52.0

### ISSUE THREAD

**Issue by jaakla** [nhachicha/android-native-dependencies#9]
I get in two machines (dev and build server):

```
Build failed :|
Unable to initialize project data from Gradle build file:

A problem occurred evaluating project ':com.nutiteq.advancedmap3'.
> com/nabilhachicha/nativedependencies/NativeDependenciesPlugin : Unsupported major.minor version 52.0
```

Same works in another machine (with Java 1.8). Perhaps the machines have older JDK, and your build is with newer JDK? Can you make the plugin to work even if developer has older Java versions?

**Comment by nhachicha** [comment]
Hi @jaakla 

Yep, the build should use the cross compilation options to target 1.7. 
can you send a PR pls?

Cheers,

**Comment by jt4uc** [comment]
Hi, I am using Android Studio and I have set my java compiler to both 1.7 and 1.8, but I still get the problem. I have tried to do cross compilation to target 1.7, but that did not work either. How would you recommend I resolve the error? Thanks!

**Comment by nhachicha** [comment]
Hi @jt4uc @jaakla 

should be fixed in `0.1.3` please update your conf

Cheers,

**Comment by jaakla** [comment]
works for me now
